.circleci/config.yml

```diff
@@ -187,6 +187,9 @@ workflows:
       - test-e2e-chrome-mv3:
           requires:
             - prep-build-test-mv3
+      - test-e2e-chrome-vault-decryption:
+          requires:
+            - prep-build
```

Comment:
Should this be `prep-build-test`?

``` suggestion
            - prep-build-test

Reply:
No, this is intentionally run against the prod build, as we need to use the real browser storage instead of the _fake_ one that is mocked against the `FixtureServer`